README.md

```diff
@@ -1,57 +1,18 @@
-# Northcoders House of Games API
+# NC Backend Project
 
-## Background
+## Project Trello
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+https://trello.com/b/zlUxNoND/be-nc-games
```

Comment:
You don't need to share the trello to the user - this is for you to manage the project but not useful for anyone else